### PR TITLE
allow passing in multiple regions as string json

### PR DESCRIPTION
**Pull request by markgllin**
allow passing in multiple regions as string json to enable deployment to more than 1 region.

### CODE REVIEW

File: .github/workflows/publish-and-deploy.yml [hunk 1]
```diff
@@ -67,6 +67,10 @@ jobs:
   build-and-publish:
     name: 'Build & Publish (${{ inputs.ecs_service_name }})'
     runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        include:
+          ${{ fromJson(inputs.region_matrix) }}
```
Comment (davegallant) [inline .github/workflows/publish-and-deploy.yml:73]: This is nice, we can also pass this into terraform as well.

File: README.md [hunk 2]
```diff
@@ -18,7 +18,7 @@ jobs:
     name: "ECR-ECS"
     uses: rewindio/github-action-ecs-deploy/.github/workflows/publish-and-deploy.yml@v0
     with:
-      aws_region: ca-central-1
+      region_matrix: '[ { region: "us-east-1" }, { region: "us-east-2" } ]'
```
Comment (davegallant) [inline README.md:21]: I don't think we'll need to pass in anything else yet? Most will just be tfvars.

File: .github/workflows/publish-and-deploy.yml [hunk 3]
```diff
@@ -3,8 +3,8 @@ name: Terraform Apply
 on:
   workflow_call:
     inputs:
-      aws_region:
-        description: "The AWS Region to deploy to"
+      region_matrix:
```
Comment (davegallant) [inline .github/workflows/publish-and-deploy.yml:6]: Maybe call it deploy_matrix since we might add different keys later?

Comment (davegallant) [reply .github/workflows/publish-and-deploy.yml:6]: Thought not sure if anythng else will ever be added?

Comment (markgllin) [reply .github/workflows/publish-and-deploy.yml]: i think it's still a good suggestion to make it more robust for future changes

**Review by davegallant — APPROVED** [review]

**Pull request merged**